quixstreams/sinks/influxdb_v3.py

```diff
@@ -0,0 +1,158 @@
+from itertools import islice
+from typing import Optional, Iterable, TypeVar
+
+try:
+    import influxdb_client_3
+    from influxdb_client_3 import InfluxDBClient3, WritePrecision, WriteOptions
+    from influxdb_client_3.write_client.client.write_api import WriteType
+
+    _influx_client_installed = True
+except ImportError:
+    _influx_client_installed = False
```

Comment:
I don't think we should do that. We decided against it with protobuf and avro serdes. It should raise at import time if dependencies are missing and we should take care this isn't imported anywhere it's not necessary.